scripts/build_virtualenv.sh

```diff
@@ -44,6 +44,7 @@ fi
 
 # If we compile for Python 3, we want to install a newer version since the version on the DAS6 is outdated.
 if [ ! -e ~/python3/bin/python3 ]; then
+    echo "Installing Python 3.9
```

Comment:
**Issue:** Is this missing an ending `"`?

Reply:
Oops... Fixed!